src/main/java/com/raoulvdberge/refinedstorage/apiimpl/util/Comparer.java

```diff
@@ -20,6 +21,11 @@ public boolean isEqual(@Nullable ItemStack left, @Nullable ItemStack right, int
         if (validity == EnumActionResult.FAIL || validity == EnumActionResult.SUCCESS) {
             return validity == EnumActionResult.SUCCESS;
         }
+        
+        if(IntegrationForestry.isLoaded()) {
+        	ItemStack[] items = {left, right};
+        	flags = IntegrationForestry.isItem(flags, items);
```

Comment:
Changing the flag here seems kinda off. I would propose adding a method in `IntegrationForestry` along the lines of `sanitise`, where it removes then tags that need to be removed without changing the comparison flags. The this if would be looking more like this:

```java
if (IntegrationForestry.isLoaded()) {
    if (IntegrationForestry.isBee(left) {
        left = IntegrationForestry.sanitise(left.copy());
    }
    if (IntegrationForestry.isBee(right)) {
        right = IntegrationForestry.sanitise(right.copy());
    }
}
```

In this approach `sanitise` could be added to or changed without changing this code.